projects/gloo/pkg/translator/clusters.go

```
@@ -102,6 +102,8 @@ func (t *translatorInstance) initializeCluster(
 		ConnectTimeout:            ptypes.DurationProto(ClusterConnectionTimeout),
 		Http2ProtocolOptions:      getHttp2options(upstream),
 		IgnoreHealthOnHostRemoval: upstream.GetIgnoreHealthOnHostRemoval().GetValue(),
+		RespectDnsTtl:             upstream.GetRespectDnsTtl().GetValue(),
+		DnsRefreshRate:            upstream.GetDnsRefreshRate(),
```

We should add the check here and report an error - adding it in update_upstream_spec will hide the failure to set as it doesn't report incorrect config, it just silently ignores it.